src/components/pages/AppSubscription/AppSubscriptionDetailOverlay/index.tsx

```diff
@@ -155,17 +177,19 @@ const AppSubscriptionDetailOverlay = ({
     )
   }
 
+  const getTechnicalName = () => {
+    if (isAppSubscription) return ''
+    else return t('content.appSubscription.detailOverlay.technicalNameInfo')
+  }
```

Comment:
better write as

`const getTechnicalName = () => isAppSubscription ? '' : t('content.appSubscription.detailOverlay.technicalNameInfo')`

Reply:
@oyo I used this to avoid code smell in which we can't use ternary operators more than 15 times. That's the reason I called function and return the value.